reddit_about/public/static/css/about.less

```diff
@@ -1064,3 +1065,61 @@
         }
     }
 }
+
+#about-values {
+
+    .container {
+        width: 950px;
+    }
+
+    .image-container {
+        width: 300px;
+        float: left;
+        box-sizing: border-box;
+        padding: 2em 1em;
+    }
+
+    .angel-snoo-graphic {
+        background: url('../about/angelic-alien.png');
+        width: 276px;
+        height: 913px;
+    }
+
+    .core-values-container {
+        float: right;
+        width: 650px;
+        box-sizing: border-box;
+        padding: 2em 1em 0em 1.5em;
+    }
+
+    ul.core-values {
+
+        h3 {
+            font-size: 1.3em;
+            margin-bottom: 0.7em;
+        }
+
+        ul.value-detail {
+            margin-left: 2em;
+            margin-bottom: 2em;
+            list-style: disc;
+            color: @lightgray;
+
+            li {
+                font-size: 0.9em;
+                line-height: 1.5em;
+                padding-bottom: 0.2em;
+            }
+        }
+    }
+}
+
+@media (-webkit-min-device-pixel-ratio: 1.25), not all, (min-resolution: 120dpi) {
```

Comment:
Hm - I'm not certain what this `not all` does in this context.

Also: We can use the `hdpi-bg-image` mixin for this now. :)